temporian/implementation/numpy/operators/tests/lag/lag_test.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2021 Google LLC.
```

Comment:
I think we agreed on dumping test scripts directly into the `tests` directory instead of creating a subdirectory (`/lag` in this case)